ts/Series/Dumbbell/DumbbellPoint.ts

```diff
@@ -116,6 +115,19 @@ class DumbbellPoint extends AreaRangePoint {
     }
 }
 
+/* *
+ *
+ *  Prototype properties
+ *
+ * */
+interface DumbbellPoint{
+    pointSetState: typeof AreaRangePoint.prototype.setState;
+}
+
+extend(DumbbellPoint.prototype, {
+    pointSetState: AreaRangePoint.prototype.setState
+});
+
```

Comment:
DumbbellPoint is extending now AreaRangePoint, so you can actually simply delete it. It was in the factory, because there all point prototypes inherited from the base Point class.

Reply:
AreaRangePoint does not have a `pointSetState` as far as I can see? But we could just call super.setState in the dumbbellPoint setstate, which would be cleaner.

Reply:
(which I now remember becomes problematic, as LollipopPoint overwrites pointSetState again)

Reply:
Oh, my mistake. It is a different name. I like your idea, but that is kind of refactoring outside of the task. So just ignore my request - it is nonsense. 😅